### PR TITLE
changed server signature

### PR DESCRIPTION
**Pull request by praj1001**
## Summary
Provide a detailed description of all the changes present in this pull request.

## Additional Context
Add any additional context about the problem here. 
- [ ] Root cause and the steps to reproduce. (If applicable)
- [ ] Thought process behind the implementation.

## Related Issues (if any)
Mention any related issues or pull requests.

## Checklist
- [ ] 🟢 Spec tests.
- [ ] 🟢 Acceptance tests.
- [ ] Manually verified. (For example `puppet apply`)

### CODE REVIEW

File: manifests/vhost.pp [hunk 1]
```diff
@@ -2368,7 +2368,7 @@
   concat::fragment { "${name}-serversignature":
     target  => "${priority_real}${filename}.conf",
     order   => 90,
-    content => template('apache/vhost/_serversignature.erb'),
+    content => epp('apache/vhost/_serversignature.epp'),
```
Comment (smortex) [inline manifests/vhost.pp:2371]: The file has no substitution, does it make sense to use a template rather than a file?

Comment (dinesh-gujar) [reply manifests/vhost.pp:2371]: we tried removing but it was failing the acceptance so created the file

Comment (ekohl) [reply manifests/vhost.pp:2385]: I think @smortex suggested to use `file()` instead of `epp()`. Then store it in `files/` instead of `templates`.

Comment (dinesh-gujar) [reply manifests/vhost.pp:2388]: sure will try that

**Comment by Ramesh7** [comment]
Checked with author and the PR is no more needed, hence closing the same.

**Pull request closed without merging**